xml/art_suseconnect_tracking.xml

```diff
@@ -0,0 +1,405 @@
+   and let you filter out registered inactive hosts. If your hosts are registered against RMT or
+   &suma;, these registration servers forward the received information to SCC.
+  </para>
+
+  <sect2 xml:id="identifying-inactive-systems-with-scc">
+   <title>Identifying inactive systems with SCC</title>
+   <para>
+    In the SCC Web user interface, you can limit the list of subscriptions to see only inactive
+    hosts.
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Log in to SCC at <link xlink:href="https://scc.suse.com"/> and select your organization in the
+      left pane.
+     </para>
+    </step>
+    <step>
+     <para>
+      Select <guimenu>Systems</guimenu> from the top menu to list all registered hosts by default.
+     </para>
+    </step>
+    <step>
+     <para>
+      Select <guimenu>Inactive</guimenu> from the top left drop-down list.
+     </para>
+     <figure>
+      <title>Drop-down list with a filter for inactive hosts</title>
+      <mediaobject>
+       <imageobject role="fo">
+        <imagedata fileref="scc-inactive-filter.png" width="45%"/>
+       </imageobject>
+       <imageobject role="html">
+        <imagedata fileref="scc-inactive-filter.png" width="45%"/>
+       </imageobject>
+      </mediaobject>
+     </figure>
+    </step>
+    <step>
+     <para>
+      If you have sufficient privileges and your inactive host is registered directly with SCC, you
+      can de-register it by clicking the three dots on its right and selecting
+      <guimenu>De-register</guimenu>.
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="identifying-inactive-systems-with-rmt">
+   <title>Identifying inactive systems with RMT</title>
+   <para>
+    By using the <command>rmt-cli purge</command> command, you can view hosts that have not
+    contacted the RMT server since a given date in the past:
+   </para>
+<screen>
+&prompt.root;rmt-cli systems purge --before 2021-06-16
++------------+----------+---------------------+---------------------+----------+
+| Login      | Hostname | Registration time   | Last seen           | Products |
++------------+----------+---------------------+---------------------+----------+
+| SCC_c5b0.. | 6e485e48b| 2021-06-11 13:38:07 | 2021-06-11 13:52:01 | SLES/15..|
+| SCC_5fcf.. | node52   | 2021-06-15 13:29:24 | 2021-06-15 13:31:25 | SLES/15..|
++------------+----------+---------------------+---------------------+----------+
+Do you want to delete these systems? (y/n) y
+Purged systems that have not contacted this RMT since 2021-06-16.
+</screen>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="system-data-gathered-by-scc">
+  <title>System data gathered by SCC</title>
+
+  <para>
+   When a system is registered directly via SCC, or its registration information is forwarded by RMT
+   or &suma;, SCC collects the following information:
+  </para>
+
+  <table>
+   <title>System attributes collected by SCC</title>
```

Comment:
Do you think we should add a description of each field, or is it self-evident?

Reply:
I guess its self-explanatory